sound/soc/sof/intel/hda-dai.c

```diff
@@ -466,18 +468,28 @@ static int ipc4_hda_dai_trigger(struct snd_pcm_substream *substream,
 
 	w = snd_soc_dai_get_widget(dai, substream->stream);
 	swidget = w->dobj.private;
+	pipe_widget = swidget->pipe_widget;
+	pipeline = pipe_widget->private;
 
 	switch (cmd) {
 	case SNDRV_PCM_TRIGGER_START:
 	case SNDRV_PCM_TRIGGER_PAUSE_RELEASE:
 		snd_hdac_ext_stream_start(hext_stream);
+		ret = sof_ipc4_set_pipeline_state(sdev, pipe_widget->instance_id,
```

Comment:
do we need to paused it for PAUSE_RELEASE case ?  The stream is in PAUSE_PUSH state and already in paused state

Reply:
fixed now @RanderWang